app/src/debug/AndroidManifest.xml

```diff
@@ -19,6 +19,10 @@
     <application
             tools:replace="android:name"
             android:name="org.mozilla.fenix.DebugFenixApplication">
+
+        <activity
+            android:name="androidx.activity.ComponentActivity"
```

Comment:
This definetly shouldn't be needed but although we're using Deps.androidx_compose_ui_test_manifest it seems the instrumentation can't find ComponentActivity.
Will have to look more in how the manifests are merged.